sanic/blueprints.py

```diff
@@ -38,11 +38,17 @@ def __init__(
         version=None,
         strict_slashes=False,
     ):
-        """Create a new blueprint
+        """
+        In *Sanic* terminology, a **Blueprint** is a logical collection of
+        URLs that perform a specific set of tasks which can be identified by
+        a unique name.
 
         :param name: unique name of the blueprint
         :param url_prefix: URL to be prefixed before all route URLs
-        :param strict_slashes: strict to trailing slash
+        :param host: IP Address of FQDN for the sanic server to use.
+        :param version: Blueprint Version
+        :param strict_slashes: Enforce the API urls are requested with a
+            training */*
```

Comment:
Oops, minor typo here: `trailing`

Reply:
Oh damn. `training` makes no sense in that context. How silly of me. 😝

Reply:
Don't worry, that's why we can review things ... And fix them later if something passes by. Nevertheless, that's an impressive work there!